config.cfg

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+set TOGGLE_CONSOLE `
+set START_GAME Return
+set STOP_GAME Escape
+set TOGGLE_HUD F1
+set SCREENSHOT F2
+set TOGGLE_DEBUG_OVERLAY F3
+set TOGGLE_DEBUG_GRID F4
+set QUICK_SAVE F5
+set QUICK_LOAD F9
+set TOGGLE_PROFILER F12
+set TOGGLE_BLENDING Space 
+set TOGGLE_CONSTRUCT_MODE m
+set TOGGLE_UNIT_DEBUG p
+set TRAIN_OBJECT t
+set ENABLE_BUILDING_PLACEMENT y
+set DISABLE_SET_ABILITY Ctrl z
+set SET_ABILITY_MOVE Ctrl x
+set SET_ABILITY_GATHER Ctrl c
+set SET_ABILITY_GARRISON g
+set SPAWN_VILLAGER v
+set KILL_UNIT Delete
+set BUILDING_1 q
+set BUILDING_2 w
+set BUILDING_3 e
+set BUILDING_4 r
+set BUILDING_5 Ctrl q
```

Comment:
I choosed to use ` ` as separator since it was the only (or one of the few) character that you have to write. I was going to use `+` has you suggested but I didn't like the idea of having `Ctrl++`